lib/ExpandablePicker/ExpandablePicker.js

```diff
@@ -254,7 +254,7 @@ module.exports = kind(
 	*/
 	rendered: function () {
 		ExpandableListItem.prototype.rendered.apply(this, arguments);
-		if (!this.$.drawer.renderOnShow) this.isDrawerRendered = true;
+		if (this.getOpen()) this.isDrawerRendered = true;
```

Comment:
Although its default value is true, user can set it false when user instantiates it.
So, I think @sbcho 's suggestion has worth to consider.